### DISCUSSION TITLE
Run with custom `catalog.json` and `collections.json`

### DISCUSSION THREAD

**Discussion by iliion** [stac-utils/stac-fastapi#353] [Q&A] [answered]
Dear all, 
i have a simple question. 
It is not clear to me how to add my own `catalog.json` and `collections.json` files.

Could someone give me a hint?

**Comment by gadomski** [comment, marked as answer]
By `catalog.json`, I assume you mean the [landing page](https://github.com/radiantearth/stac-api-spec/tree/main/core#example-landing-page-for-stac-api---core)? If so, you can customize title, description, etc when you create your API (e.g. here's how it's done on the Planetary Computer: https://github.com/microsoft/planetary-computer-apis/blob/26a67e1266c7fc1ba190bdc5406cbfa2f6855edc/pcstac/pcstac/main.py#L57-L74). If you need additional customization, you can override the `landing_page` method on your client: https://github.com/stac-utils/stac-fastapi/blob/3db04da73b6a65367e4a94f7262067c61650225d/stac_fastapi/types/stac_fastapi/types/core.py#L540

As for adding your own Collections, you can use the API itself to load collections and itmes; check out this script in the repo for an example: https://github.com/stac-utils/stac-fastapi/blob/master/scripts/ingest_joplin.py. If you're using [pgstac](https://github.com/stac-utils/pgstac), you could also use the `pypgstac` loader: https://github.com/stac-utils/pgstac/blob/main/docs/src/pypgstac.md#bulk-data-loading.

HTH

**Comment by iliion** [comment]
@gadomski Thanks for the answer but the issue has been resolved. I appreciate the usefulness of the link to the repository
